extensions/wikia/FliteTag/FliteTagController.class.php

```diff
@@ -75,6 +76,15 @@ public function fliteAdUnit() {
 		$this->response->setTemplateEngine( WikiaResponse::TEMPLATE_ENGINE_MUSTACHE );
 	}
 
+	public function fliteAdUnitMobile() {
+		$this->setVal( 'guid', $this->getVal( 'guid' ) );
+		$this->setVal( 'width', $this->getVal( 'width' ) );
+		$this->setVal( 'height', $this->getVal( 'height' ) );
```

Comment:
What do you think about extracting setting `guid`, `width` and `height` to a separate method (i.e. `setSkinSharedVariables()`) and re-use it in `fliteAdUnitDesktop()` and `fliteAdUnitMobile()`? Over-engineering? :japanese_ogre: